src/devices/gps_skytraq_s1216_sup500f8.c

```diff
@@ -669,6 +668,8 @@ gps_status_t GPS_device_init(uint8_t sampleRate, struct Serial *serial)
     size_t gps_init_status = GPS_STATUS_NOT_INIT;
 
     while(attempts-- && gps_init_status == GPS_STATUS_NOT_INIT) {
+        /* Perform low level init / reset of GPS device */
+        gps_device_lld_init();
```

Comment:
Does this cause a power cycle?  If so be careful of the Mk2 bug RE delay needed before GPS init.